### ISSUE TITLE
Doesn't work in nodejs 0.12

### ISSUE THREAD

**Issue by dfilatov** [nodules/luster-guard#8]
`luster-guard` version 0.4.0
`luster` version 0.5.8
```
.../node_modules/luster-guard/node_modules/gaze/lib/platform.js:125
 PathWatcher.closeAllWatchers();
             ^
TypeError: Cannot read property 'closeAllWatchers' of null
```

**Comment by narqo** [comment]
Fixed \w #9 and released with v0.5.0

**Comment by dfilatov** [comment]
awesome!
